tests/vm_test.go

```diff
@@ -169,7 +170,28 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		type vmiBuilder func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume)
 
 		newVirtualMachineInstanceWithFileDisk := func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume) {
-			return tests.NewRandomVirtualMachineInstanceWithFileDisk(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), testsuite.GetTestNamespace(nil), corev1.ReadWriteOnce)
+			sc, foundSC := libstorage.GetRWOFileSystemStorageClass()
+			Expect(foundSC).To(BeTrue(), "Filesystem storage is not present")
```

Comment:
Previously it as skipped if the storage class wasn't found. Are we sure that any cluster running this test has a sc?

Reply:
Yes, this is part of the effort of removing skips on tests and using Expect on them so that we wont skip necessary tests. 
https://github.com/kubevirt/kubevirt/issues/11632

Reply:
lines 373-375